### PR TITLE
support static images

### LINKED ISSUE #1: use static image.
Is it possible to use static image in the app instead of loading it from a url/network?

Definitely want to use this component in my app. Thanks for this. :)

### PR DESCRIPTION
**Pull request by chirag04**
fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,11 @@ Showing the ActivityView is as simple as calling:
 ActivityView.show({
   text: "Text you want to share",
   url: "URL you want to share",
-  imageUrl: "Url of the image you want to share/action"
+  imageUrl: "Url of the image you want to share/action",
+  image: "name of the image in the app bundle"
```
Comment (naoufal) [inline README.md]: Nitpick, but can you capitalize `Name`.

File: README.md [hunk 2]
```diff
@@ -36,9 +36,11 @@ Showing the ActivityView is as simple as calling:
...
 });
 ```
+Note: Only one out of image and imageUrl should be given. If both are given `image` will be used.
```
Comment (naoufal) [inline README.md]: Do you mind updating this note with:

```md
_Note: Only provide one image type to the options argument.  If multiple image types are provided, `image` will be used._
```

**Comment by naoufal** [comment]
Aside from the minor documentation comments.  It looks good to me.  :rocket:

**Comment by naoufal** [comment]
#### Food for Thought™:

The `imageUrl` or `image` situation is making me think the API could be updated to something like this in the future:

```js
text: 'foo',
media: {
  type: 'image',
  source: 'your-image'
}
```
Which would look like this for an imageUrl:
```js
text: 'foo',
media: {
  type: 'imageUrl',
  source: 'http://foo.com/image.jpg'
}
```
And eventually, something like this for video:
```js
text: 'foo',
media: {
  type: 'videoUrl',
  source: 'http://foo.com/image.mp4'
}
```

@chirag04 What do you think?

**Comment by chirag04** [comment]
Fixed the readme. 

As for the new api, i definitely like this new api. Much more flexible and extensive. This can be part of the next major version bump.

**Pull request merged**